src/components/mode-tools/mode-tools.jsx

```diff
@@ -89,6 +89,8 @@ const ModeToolsComponent = props => {
     case Modes.BIT_BRUSH:
         /* falls through */
     case Modes.BIT_LINE:
+        /* falls through */
+    case Modes.BIT_RECT:
```

Comment:
Seems a bit strange to have this here. It results in an unlabeled, non-functional input in the mode tools bar. Should this PR just leave out the changes in this file, leaving the mode tools area blank?

Reply:
I'll leave it out until drawing rectangle outlines is added.